heapsort/heapsort.py

```diff
@@ -0,0 +1,25 @@
+def s(l):
+    #size
+    n = len(l)
+    print("n:",n)
+    print("range:", range(n,-1,-1))
```

Comment:
this print statement outputs `range: range(n, -1, -1)` for whatever value of n so is not providing any useful information since we already printed out `n`